### ISSUE TITLE
Add korok seeds discovery status

### ISSUE THREAD

**Issue by ttimasdf** [timsama/botw-save-file-mapper#3]
Seen from [marcroblendo's save game editor](https://github.com/marcrobledo/savegame-editors), but I don't know if the saves are compatible.

Reference data:

https://web.archive.org/web/20200419045640/https://www.marcrobledo.com/savegame-editors/zelda-botw/zelda-botw.data.js

**Comment by ttimasdf** [comment]
The logic behind Korok seeds and related NPC is so complicated 😵 I'll check it out with my spare time.

**Comment by timsama** [comment]
Yeah, this is the part that I was dreading doing all by myself, haha.
